### PR TITLE
Cleanup Why might I have this debt additional info component

### PR DESCRIPTION
**Pull request by JunTaoLuo**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

Remove unused Why might I have this debt additional info components

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/92812

## Testing done

- This PR does not introduce new behaviors and only removes dead code so ensuring existing tests pass is sufficient

## Screenshots

N/A

## What areas of the site does it impact?

- Combined Debts Portal
- Financial Status Report

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/financial-status-report/constants/deduction-codes/index.js [hunk 1]
```diff
@@ -10,210 +7,3 @@ export const deductionCodes = Object.freeze({
   '74': 'Post-9/11 GI Bill debt for tuition',
   '75': 'Post-9/11 GI Bill debt for tuition (school liable)',
 });
-
-export const renderAdditionalInfo = deductionCode => {
```
Comment (JunTaoLuo) [inline src/applications/financial-status-report/constants/deduction-codes/index.js:14]: Oops I just realized that this team generally split CDP and FSR changes into separate PRs. I did find that both `renderAdditionalInfo` and `renderWhyMightIHaveThisDebt` are unused in FSR so I removed them as well but let me know if I should revert this or do it in a separate PR.

Comment (Dr-Pongo) [reply src/applications/financial-status-report/constants/deduction-codes/index.js:14]: This is minor enough it should be fine! 🤙

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀
